<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auth</class>
 <widget class="QWidget" name="Auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.573864, y1:0.006, x2:0.551, y2:0.988636, stop:0.375 rgba(141, 206, 180, 255), stop:1 rgba(255, 255, 255, 255));
border-top-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>82</y>
     <width>29</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>PASS:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Host_name">
   <property name="geometry">
    <rect>
     <x>70</x>

     <y>30</y>

     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DB_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>108</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>myassistant</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>56</y>
     <width>30</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>USER:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Connect">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>154</y>
     <width>190</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
    border-style: outset;
    border-width: 2px;
    border-color: gray;
    padding: 4px;
	background-color: rgb(141, 206, 180);
	border-radius: 10px;


}

QPushButton:pressed {
    border-style: inset;
    background-color: #c0c0c0;
}
</string>
   </property>
   <property name="text">
    <string>Подключится к ТИЧА</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>214</y>
     <width>58</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pic.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="Pass_value">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>82</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>gfhjkmlkz,fpslfyys[1!</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>108</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>DB_NAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="User_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>56</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>root</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>11</y>
     <width>97</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключение к БД</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>30</y>
     <width>31</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>HOST:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../logo.qrc"/>
 </resources>
 <connections/>
</ui>
